_i18n/es/resources/user-guides/easiest_buy.md

```diff
@@ -1,66 +1,65 @@
+Después de hacer clic, verás tu clave privada. Esta llave es lo que te dará acceso a tus fondos. ḂNunca compartas esta clave con nadie!
 
-### WRITE DOWN THIS KEY IMMEDIATELY!
+### ESCRIBE Y GUARDA ESTA LLAVE INMEDIATAMENTE
 
 ![image2](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/02.png)
 
-Type in your private key in the box below, and click the button.
+Escribe tu clave privada en el espacio mostrado, y haz clic en el botón
 
-On the next page, you will see your address.
+En la siguiente página, podrás ver tu dirección.
 
 ![image3](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/03.png)
 
-Copy your address to the clipboard by highlighting the whole thing and hitting ctrl+c (or edit menu, copy), or clicking the little icon next to your address. Save your address somewhere. This is how others will send Monero to you, and what you will use to deposit Monero into your account!
+Copia tu dirección en el portapapeles seleccionando toda la cadena y presionando Ctrl+C (o en Edit menu, Copy), o presionando el pequeño icono al lado de tu dirección. Guarda tu dirección en algún lugar. Con ella otros usuarios podrán enviarte Monero, y será lo que utilizarás para depositar Monero en tu cuenta.
 
-#### Step 3: Buy Monero and transfer the Monero to your new address
+#### Paso 3: Compra y transfiere Monero a tu nueva dirección
 
-Go to www.shapeshift.io . On the righthand side, of the screen, click icon under "Receive" to select Monero.
+Ve a www.shapeshift.io. En el lado derecho de la pantalla, haz clic en el icono de recibir (Receive) para seleccionar Monero.
 
 ![image5](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/05.png)
 ![image6](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/06.png)
 
-Paste your address into the field under the Monero logo. Select the "agree to terms" button, then hit "Start"
+Pega tu dirección en el campo bajo el logo de Monero. Acepta los términos marcando el botón correspondiente (agree to terms) y presiona Start.
 
 ![image7](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/07.png)
 
-In the new screen that pops up, copy the Deposit Address into your clipboard (select and hit ctrl+c or edit-copy)
+En la nueva pantalla que saldrá, copia la dirección de depósito en el portapapeles (seleccionar y Ctrl+C o editar-copiar)
 
 ![image8](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/08.png)
 
-Go back to your circle.com page, hit the "transfer" button, and paste the Bitcoin address into the field
-Enter the amount of Bitcoin you would like to spend.
+Regresa a la página circle.com, presiona el botón de transferir (transfer), y pega la dirección de Bitcoin en el campo. Ingresa la cantidad de Bitcoin que deseas invertir.
```

Comment:
"spend" - "gastar"?

Reply:
Al fin y al cabo va a gastar bitcoin para comprar monero, no?

Reply:
Técnicamente sí es gastar, ¿te parece si la cambiamos o lo dejamos así? Puse _invertir_ porque es como que, utilizarlos para luego minar y obtener Monero, o sea un tipo de inversión jeje.

Reply:
Técnicamente tienes razón. jajaja. Ahí depende del contexto. También sirve, dejala y dependiendo de la reacción de los que lean las guías más adelante podríamos saber si les gusta o no. :)

Reply:
yo diría "gastar", para no introducir un concepto que no está presente en la fuente. también se evita un debate si realmente es una inversión o no.

Reply:
I agree with **"gastar"**.

Reply:
Parece bien, pensaba en otra cosa al momento de traducir. Sin embargo qué les parece si utilizamos un concepto más general del español, como _utilizar_? Esto en caso de hacer como en otras traducciones que preferimos utilizar algo un poco distinto a la redacción original para adaptar el contexto a algo más neutral. O si se prefiere puede ser un estilo de traducción más directo y utilizar _gastar_ en este caso.

Reply:
I wouldn't use the word "utilizar" because might confuse with its definition of "use" and not spend. You are literally "spending" BTC to buy XMR. 

e.g. 
We **"use"** BTC as a media of exchange, like money, but we **"spend"** BTC to buy a movie ticket, and we also "**use**" BTC to buy a movie ticket. 


I understand the context in a different meaning. Just like @ordtrogen said in the earlier comment, that we might get into a discussion if it's an, "investment = inversión", and we literally are in a discussion if it's, "spend = gastar" or "use = utilizar". :p 
Spend - noun informal
1. an amount of money paid out.